test/lib/server/services/qualityControlFlag/QcFlagService.test.js

```diff
@@ -578,4 +580,78 @@ module.exports = () => {
             expect(fetchedFlagWithSimulationPass.simulationPasses.map(({ id }) => id)).to.have.all.members([relations.simulationPassId]);
         });
     });
+
+    describe('Delating Quality Control Flag', () => {
+        it('should fail to delete QC flag (of dataPass) when being neither owner nor admin', async () => {
```

Comment:
Why puting `(of dataPass)` between brackets? You can simply put it normally
Same for all the rest

Reply:
applied